app/models/exhibit_finder.rb

```diff
@@ -44,7 +44,7 @@ def exhibit_slugs_for_search(query)
         }
       ).dig('response', 'docs') || {}
 
-      documents.map { |document| document['exhibit_slug_ssi'] }
+      documents.pluck('exhibit_slug_ssi')
```

Comment:
Holy crap. I didn't know there was an ActiveSupport Array method that did this.

Reply:
yeah...it was news to me too